src/benchmarks/micro/corefx/System.Text.Json/Utf8JsonReader/Perf.Comments.cs

```diff
@@ -11,16 +11,14 @@
 namespace System.Text.Json.Tests
 {
     [BenchmarkCategory(Categories.CoreFX, Categories.JSON)]
-    public partial class Utf8JsonReaderCommentsTests
+    public partial class Utf8JsonReader_Comments
```

Comment:
Maybe would be better to put Utf8JsonReader somewhere in the namespace and making a class name just `CommentsTests`

Reply:
I thought about that, but then the test namespace doesn't follow the API namespace structure. I believe I was told that the pattern is `<API name space>.Tests`.